client/css/style.css

```diff
@@ -1614,8 +1624,8 @@ kbd {
 	animation-fill-mode: forwards;
 	-webkit-animation-timing-function: ease-in;
 	animation-timing-function: ease-in;
-	-webkit-animation-delay: .4s;
-	animation-delay: .4s;
+	-webkit-animation-delay: .1s;
+	animation-delay: .1s;
```

Comment:
Yes, and again there are good reasons why we added a delay.

Anyway, I feel like we are going in circles here. This is my final review for this PR: **please revert the animation delay change and you should get my 👍.** Thanks!